.rubocop.yml

```diff
@@ -0,0 +1,10 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
```

Comment:
You can add here rubocop configuration and try it out ;)

Reply:
Gonna stick with the defaults for the moment, but thanks :D